.github/workflows/reset-environment.yml

```
@@ -0,0 +1,219 @@
+name: Reset Environment
```

Also change the file name.

```suggestion
name: Reset Azure Environment
```